src/static/js/modules/youtube.js

```diff
@@ -0,0 +1,177 @@
+/* ==========================================================================
+   Youtube Initialization
+   Used on at least `/the-bureau/`.
+   ========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+
+function init() {
+  return $( '.youtube-player_container' ).createYoutube();
+}
+
+// Youtube constructor.
+// @param element {Element} DOM Element.
+// @param options {Object} Object used to customize Youtube.
+// Returns {Object} An Youtube instance.
+function Youtube( element, options ) {
+  this.$element = $( element );
+  this.options = $.extend( true, {}, this.defaults, options );
+  this.init();
+}
+
+// Youtube methods and shared properties
+Youtube.prototype = {
+
+  constructor: Youtube,
+
+  defaults: {
+     playerOptions : {
+          height: '410',
+          width: '640',
+          videoId: 'en0Iq8II4fA',
+          playerVars: { 
+              'autoplay': 1, 
+              'suggestedQuality': 'highres'
+          },
+          events: {
+              'onReady': $.noop,
+              'onStateChange': $.noop
+          }
+      },
+      playerInitialized : false
+  },
+
+  _bindAll: function() { 
+    var self = this;
+    var proxy = $.proxy;
+
+    // TODO: Swap this out for Function.bind
+    // and Object.keys. 
+    $.each( this, function( key, _function ){
+        if ( $.isFunction( _function ) ) {
+            self[key] = proxy( _function, self );
+        }
+    });
+  },
+
+  _embedScript: function() {
+      window.onYouTubeIframeAPIReady = this.initPlayer;
+
+      var tag = document.createElement('script');
+      tag.src = "http://www.youtube.com/iframe_api";
+      var firstScriptTag = document.getElementsByTagName('script')[0];
+      firstScriptTag.parentNode.insertBefore( tag, firstScriptTag );
```

Comment:
So for our proposes youtube-node is unnecessary. We are really only using the api for three events. So I am going to use jsLoader.loadScript( 'http://www.youtube.com/iframe_api' ) just be inline with other implementations. The Google Tag Manager is actually inserting the api into the dom so the _embedScript currently isn't running. I only included the code in case the api didn't load for some reason.